<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_group_window</class>
 <widget class="QMainWindow" name="main_group_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 198, 198);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_all_groups">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название группы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Куратор группы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество человек</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_save_change">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Сохранить изменения
в таблице</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_group">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Удалить группу</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_out_all_groups">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Вывести всю информацию
в таблицу</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_add_group_form">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Добавить новую группу</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_choice_filter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Название группы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Куратор</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edit_filter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Выберите критерий в выпадающем списке</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_out_criteria_group">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Вывести 
по критерию</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>355</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Критерий выбора</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_change_curator">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: rgb(133, 125, 86); 
color: White; border-radius: 10px;}
QPushButton:pressed {background-color:rgb(43, 252, 50) ; }</string>
    </property>
    <property name="text">
     <string>Изменить куратора
у группы</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
